docs/development/astropy-package-template.rst

```diff
@@ -59,7 +59,7 @@ by ``CHANGES.md`` in the instructions.
 
    All following instructions will assume you have ``pyproject.toml``.
    If you do not use ``pyproject.toml`` yet, please see
-   https://docs.astropy.org/en/v4.0.1/development/astropy-package-template.html
+   https://docs.astropy.org/en/v4.0.x/development/astropy-package-template.html
```

Comment:
I'm not sure why we should expose the branch docs, so what about keeping it linked to "LTS"?
```suggestion
   https://docs.astropy.org/en/lts/development/astropy-package-template.html
```

Reply:
But what if when the LTS changes to 5.0 in the future?

Reply:
at that point, everyone should just use `pyproject.toml` rather than look at many years old docs. But this is not a strong preference, so I'm ok with what you had in the PR.

Reply:
I would actually prefer to keep it as-is, because this way, it is clear that someone has to refer to 4.0.x doc rather than whatever is the current LTS.